src/index.ts

```diff
@@ -31,39 +39,73 @@ export class TestxServer {
   private queries: { [id: string]: any };
   private mutations: { [id: string]: any };
 
+  /**
+   * Create a TestxServer.
+   * @constructor
+   * @param {String} schema - The GraphQL data model
+   */
   constructor(schema: string) {
     this.schema = schema;
   }
 
+  /**
+   * Starts the server.
+   * @async
+   */
   public async start() {
     await this.bootstrap();
     const port = await getAvailablePort();
     this.server = this.expressServer.listen({ port });
     this.serverUrl = `http://localhost:${port}/graphql`;
   }
 
+  /**
+   * Stops the server to receive requests, but keeps the generated GraphQL backend
+   * and database connection.
```

Comment:
Myabe worth to mention that it's possible to restart the server using the `start()` method